examples/aws/iam_password_length.rego

```diff
@@ -0,0 +1,37 @@
+package rules.iam_password_length
+
+import data.fugue
+
+# We need to write an advanced rule so we can take care of the case when there's
+# no policy at all.
+resource_type = "MULTIPLE"
+
+# All password policies in the input.
+password_policies = fugue.resources("aws_iam_account_password_policy")
+
+# Is there at least one password policy in the input?
+exists_password_policy {
+  _ = password_policies[_]
+}
+
+# Is the password policy in compliance?
```

Comment:
Maybe add a comment to clarify that the user can customize the minimum password length here? That way it'll be easy to find.